modules/apps/layout/layout-reports-web/test/js/components/NotConfigured.test.js

```diff
@@ -74,4 +74,27 @@ describe('NotConfigured', () => {
 			getByText('configure-google-pagespeed-to-run-a-page-audit')
 		).toBeInTheDocument();
 	});
+
+	it('renders the proper image', () => {
+		const {getByAltText} = render(
+			<StoreContextProvider
+				value={{
+					data: {
+						configureGooglePageSpeedURL: 'validURL',
+						imagesPath: 'imagesPath',
+					},
+				}}
+			>
+				<NotConfigured />
+			</StoreContextProvider>
+		);
+
+		const image = getByAltText(
+			Liferay.Language.get(
```

Comment:
Yo don't need to use `Liferay.Language.get` in tests, you can put just the key 🙂

Reply:
Got it! :)